coalib/output/Logging.py

```diff
@@ -1,10 +1,26 @@
 from datetime import datetime
+from collections import defaultdict
 import json
 import io
 import logging
 import logging.config
 
 
+class CounterHandler(logging.Handler):
+    """
+    A logging handler which counts the number of calls
+    for each logging level.
+    """
+    level_called = None
+
+    def __init__(self, *args, **kwargs):
+        super(CounterHandler, self).__init__(*args, **kwargs)
+        CounterHandler.level_called = defaultdict(lambda: 0)
```

Comment:
@NiklasMM Hi :)

**[0]** Since we only support Python 3, `super().__init__(...)` is enough here

**[1]** Why initializing `level_called` in `__init__` when it is always used on class level? Just initialize it directly on defining it above. Then `__init__` also becomes completely obsolete. And **[0]** becomes so too :) And **most important**, then `level_called` is not re-initialized when `CounterHandler` is intantiated more than once for whatever reason...

**CC** @jayvdb